core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixTextToSpeech.kt

```diff
@@ -160,16 +163,25 @@ class KiwixTextToSpeech internal constructor(
       AudioManager.AUDIOFOCUS_GAIN
     )
     Log.d(TAG_KIWIX, "Audio Focus Requested")
+    abandonAudioFocus()
```

Comment:
I doubt we want to immediately abandon the audio focus request. This is probably meant to be in shutdown.

With this our focus change code would not run

```
focusChange -> {
      Log.d(TAG_KIWIX, "Focus change: " + focusChange);
      if (tts.currentTTSTask == null) {
        tts.stop();
        return;
      }
      switch (focusChange) {
        case (AudioManager.AUDIOFOCUS_LOSS):
          if (!tts.currentTTSTask.paused) tts.pauseOrResume();
          pauseTTSButton.setText(R.string.tts_resume);
          break;
        case (AudioManager.AUDIOFOCUS_GAIN):
          pauseTTSButton.setText(R.string.tts_pause);
          break;
      }
    }
```

Is audio focus something you are familiar with? It is basically how the system negotiates who gets to use the speakers so that you don't start playing spotify and a youtube video at the same time.